### PR TITLE
sinkv2(ticdc): improve logs for table sink

### PR DESCRIPTION
**Pull request by hi-rustin**
Issue Number: ref https://github.com/pingcap/tiflow/issues/5928
### What is changed and how it works?
improve logs for table sink. Print changefeedID and checkpointTs.

#### Tests 

 - No code

##### Will it cause performance regression or break compatibility?
No
##### Do you need to update user documentation, design documentation or monitoring documentation?
No
### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/sinkv2/tablesink/table_sink_impl.go [hunk 1]
```diff
@@ -108,32 +112,49 @@ func (e *eventTableSink[E]) UpdateResolvedTs(resolvedTs model.ResolvedTs) error
...
+	log.Info("Stopping table sink",
+		zap.String("namespace", e.changefeedID.Namespace),
+		zap.String("changefeed", e.changefeedID.ID),
+		zap.Int64("tableID", e.tableID),
+		zap.Uint64("checkpointTs", stoppingCheckpointTs.Ts))
 	err := e.progressTracker.close(ctx)
 	if err != nil {
-		log.Error("Failed to stop table sink", zap.Error(err), zap.Int64("tableID", e.tableID),
+		failedCheckpointTs := e.GetCheckpointTs()
+		log.Error("Failed to stop table sink", zap.Error(err),
```
Comment (3AceShowHand) [inline cdc/sinkv2/tablesink/table_sink_impl.go:143]: please print the `err` at the last, because it may be quite long.

**Review by 3AceShowHand — APPROVED** [review]

**Review by hicqu — APPROVED** [review]

**Pull request merged**